datarequest.py

```diff
@@ -1021,31 +956,24 @@ def api_datarequest_review_submit(ctx, data, request_id):
         set_status(ctx, request_id, "reviewed")
 
         # Get parameters needed for sending emails
-        researcher_name = ""
-        researcher_email = ""
-        bod_member_emails = ""
-        rows = row_iterator(["META_DATA_ATTR_NAME", "META_DATA_ATTR_VALUE"],
-                            "COLL_NAME = '%s' AND " % coll_path
-                            + "DATA_NAME = 'datarequest.json'",
-                            AS_DICT, ctx)
-        for row in rows:
-            name = row["META_DATA_ATTR_NAME"]
-            value = row["META_DATA_ATTR_VALUE"]
-            if name == "name":
-                researcher_name = value
-            elif name == "email":
-                researcher_email = value
+        datarequest = jsonutil.read(ctx, coll_path + "/datarequest.json")
+
+        researcher = datarequest['researchers']['contacts'][0]
+
+        researcher_name = researcher['name']
```

Comment:
Is there any reason to not use researcher['name'] directly? They are only used in the mail call? Same for researcher_email.

Reply:
No. Fixed.